lib/src/Uncertainty/Distribution/Multinomial.cxx

```diff
@@ -310,6 +364,88 @@ Scalar Multinomial::computeCDF(const Point & point) const
   return std::max(0.0, std::min(1.0, value * normalizationCDF_));
 }
 
+/* Compute the probability content of an interval */
+Scalar Multinomial::computeProbability(const Interval & interval) const
+{
+  const UnsignedInteger dimension = getDimension();
+  if (interval.getDimension() != dimension) throw InvalidArgumentException(HERE) << "Error: the given interval must have dimension=" << dimension << ", here dimension=" << interval.getDimension();
+
+  // Empty interval
+  if (interval.isNumericallyEmpty()) return 0.0;
+
+  // Early exit for 1D case
+  if (dimension == 1)
+  {
+    const Scalar a = interval.getLowerBound()[0];
+    const Scalar b = interval.getUpperBound()[0];
+    if ((a > n_ + supportEpsilon_) || (b < -supportEpsilon_)) return 0.0;
+    if ((a < -supportEpsilon_) && (b > n_ + supportEpsilon_)) return 1.0;
+    return DistFunc::pBeta(n_ - floor(b), floor(b) + 1, 1.0 - p_[0]) - DistFunc::pBeta(n_ - floor(a), floor(a) + 1, 1.0 - p_[0]);
+  }
+  const Point lower(interval.getLowerBound());
+  const Point upper(interval.getUpperBound());
+  Indices a(dimension_);
+  Indices b(dimension_);
+  UnsignedInteger sigmaA = 0;
+  UnsignedInteger sigmaB = 0;
+  for (UnsignedInteger i = 0; i < dimension_; ++i)
+    {
+      a[i] = std::max(0UL, static_cast<UnsignedInteger>(std::ceil(lower[i])));
```

Comment:
max(0, uint x) will always return x

Reply:
Yep. Cast and max should be reversed. Done.